webpack.playground.config.js

```diff
@@ -63,7 +65,9 @@ module.exports = {
     ],
     historyApiFallback: true,
     headers: {
-      'Content-Security-Policy': `script-src http://localhost:${DEV_SERVER_PORT}`
+      'Content-Security-Policy': [
+        `script-src http://${HOST}:${DEV_SERVER_PORT}`
+      ].join(';')
```

Comment:
Does changing this string construction make a difference or is it just for future devs who need to add more CSP items?